manager/integration/tests/test_basic.py

```diff
@@ -125,6 +144,10 @@ def test_volume_basic(clients, volume_name):  # NOQA
     volume = client.by_id_volume(volume_name)
     assert volume["endpoint"] == DEV_PATH + volume_name
 
+    # RW test
+    assert volume_valid(volume["endpoint"])
+    volume_rw_test(volume["endpoint"])
```

Comment:
Put volume_valid() in volume_rw_test(). volume_rw_test() doesn't make sense if volume is not valid.